### DISCUSSION TITLE
Is there any way to disable this new double-click behavior?

### DISCUSSION THREAD

**Discussion by ImoutoChan** [gitextensions/gitextensions#10972] [Q&A]
Is there any way to disable this new double-click behavior? It constantly opens random windows like the commit history of a file or commits from the graph. 

At the very least use default Windows settings for double clicking, you set reaction time too slow and detect double clicking even if the second click was in a completely different place.

**Comment by ImoutoChan** [comment]
Or maybe you have some insane settings like if the second click is close enough to the first one (with 20 px) then it's double click. Please provide a way to disable it.

**Reply by pmiossec** [reply]
I'm very interested by knowing what is this "new double-click behavior" you are talking about. Because I don't see one...
Give more inputs...

**Comment by vbjay** [comment]
Double click speed is managed by Windows. Check that first. 

Use https://www.screentogif.com/ and record a gif of an example.
